src/vs/workbench/parts/files/electron-browser/views/explorerView.ts

```diff
@@ -868,11 +869,15 @@ export class ExplorerView extends TreeViewsViewletPanel implements IExplorerView
 	private getResolvedDirectories(stat: ExplorerItem, resolvedDirectories: URI[]): void {
 		if (stat.isDirectoryResolved) {
 			if (!stat.isRoot) {
-
+				const areEqual = (first: string, second: string) => isLinux ? first === second : equalsIgnoreCase(first, second);
 				// Drop those path which are parents of the current one
 				for (let i = resolvedDirectories.length - 1; i >= 0; i--) {
-					const resource = resolvedDirectories[i];
-					if (resources.isEqualOrParent(stat.resource, resource, !isLinux /* ignorecase */)) {
+					const resourceStr = resolvedDirectories[i].toString();
```

Comment:
@isidorn can you not do the same `stat.find()` call here that you use in the other place?

Reply:
@bpasero I see. I am not aware of that. Will check for `json` first.

Reply:
@sandy081 thanks

Reply:
@bpasero no I can not since the direction is different. I could if I had a `stat.findParent`